media-server/README.md

```diff
@@ -0,0 +1,58 @@
+# Gst-Inference Media Server Demo
+
+## Dependencies
+The demo uses some of the RidgeRun open source plugins. Please make sure you install and setup the following dependencies before running the demo
+
+* [GStreamer Daemon](https://developer.ridgerun.com/wiki/index.php?title=GStreamer_Daemon_-_Building_GStreamer_Daemon)
+* [GStreamer Interpipes](https://developer.ridgerun.com/wiki/index.php?title=GstInterpipe_-_Building_and_Installation_Guide)
+
+## Demo Summary
+
+The demo can switch between two origins (camera or rtsp stream) and produces 3 outputs (udp streaming, recording and display) as seen in the following image:
+
+![Alt text](rsrc/demo.png?raw=true "Title")
+
+## Demo Execution
+
+Make sure GStreamer Daemon (gstd) is up and running previous to running the media server demo on the media-server directory:
+```bash
+gstd -D
+```
+
+### Camera RTSP source video
+The demo expects a RTSP at RTSP_URI (defined at config.txt).
+To setup a camera rtsp source for the media server, the following pipeline can be used as a reference. Please adjust formats and resolutions as needed.
+
+```bash
+gst-launch-1.0 v4l2src device=/dev/video2 ! video/x-raw, width=1280, height=720, framerate=10/1, format=YUY2 ! videoconvert ! x264enc ! video/x-h264, mapping=/stream1 ! rtspsink service=5005
+```
+
+Then run the script in another terminal with:
+
+```bash
+./media_server.sh
+```
+
+You can change between sources by using the input 
+
+To change to the RTSP stream:
+```bash
+> stream
+```
+
+To change to the camera stream:
+```bash
+> camera
+```
+
+To save the recording and exit the demo
+```bash
+> exit
+```
+
+### UDP destination video
+The demo produces a UDP output stream. To receive the output video the media server generates, the following pipeline can be used as a reference. Please adjust port as needed.
+
+```bash
+gst-launch-1.0 udpsrc port=5000 ! queue  ! tsdemux ! h264parse ! avdec_h264 ! queue ! videoconvert ! xvimagesink sync=false async=false -e
```

Comment:
Always use `autovideosink` to make the pipeline portable. Also, prefer `decodebin`. Do we really need `sync=false` and `async=false`?

Reply:
Also put an example using vlc:
```bash
vlc udp://@127.0.0.1:5000
```